### PR TITLE
chore: regroup common public nginx-ingresses values

### PR DESCRIPTION
**Pull request by lemeurherve**

### CODE REVIEW

**Comment by dduportal** [comment]
@lemeurherve can you rebase this branch and fix the conflicts?

**Comment by lemeurherve** [comment]
@dduportal done

File: updatecli/updatecli.d/docker-images/404.yaml [hunk 1]
```diff
@@ -39,46 +39,18 @@ targets:
...
-      file: "config/ext_public-nginx-ingress_prodpublick8s.yaml"
-      key: "defaultBackend.image.tag"
-    scmid: default
   updatePrivateNginxIngress404TempPrivateK8s:
     name: "Update 404 docker image tag"
     kind: yaml
     spec:
       file: "config/ext_private-nginx-ingress_temp-privatek8s.yaml"
       key: "defaultBackend.image.tag"
     scmid: default
```
Comment (dduportal) [inline updatecli/updatecli.d/docker-images/404.yaml:48]:
```suggestion
```

Comment (dduportal) [reply updatecli/updatecli.d/docker-images/404.yaml:48]: (Unless I missed something?)

Comment (lemeurherve) [reply updatecli/updatecli.d/docker-images/404.yaml:48]: I didn't regrouped (yet?) the private nginx ingress config, only the public one.

**Review by dduportal — APPROVED** [review]
LGTM 👍

**Pull request merged**